src/gui/editorwidgets/qgsrelationreferencewidget.h

```diff
@@ -276,6 +275,14 @@ class GUI_EXPORT QgsRelationReferenceWidget : public QWidget
     //! unset the currently related feature
     void deleteForeignKeys();
 
+    /**
+      * Trigger save of the embedded referenced attribute form.
+      * Returns TRUE on success or if no embedded form is present.
+      *
+      * \since QGIS 3.40
+      */
+    bool saveReferencedAttributeForm();
```

Comment:
Does this need to be a slot? I think it can be a regular method

Reply:
Could be yes. I placed it more to the slot section because I thought it could be useful for future use.

It's already merged in the later releases 3.44 etc. Would you change it in the LTR now?

Reply:
If it has been merged to master already, then I would also merge this here as is.
In general I wouldn't add more slots "on reserve", it's also easy to write a lambda to connect regular methods if needed or promote to a slot later on.